majordodo-core/src/test/java/majordodo/replication/AcquireLeadershipTest.java

```diff
@@ -228,8 +228,6 @@ public void run() {
 //                log.getClusterManager().getZooKeeper().delete(zkServer.getPath() + "/leader", -1);
                 System.out.println("SET DATA AT " + zkServer.getPath() + "/leader");
 
-                broker1.getClient()
-                    .submitTask(new AddTaskRequest(0, TASKTYPE_MYTYPE, userId, taskParams, 0, 0, 0, null, 0, null, null)).getTaskId();
```

Comment:
Why did you  removed the summit?

Reply:
I think this was wrong, because here the broker is not the leader anymore and the submit should fail (there is assertion on the next line)

maybe it was flaky test
I feel comfortable about this change

Reply:
When you delete the znode the leadership is not lost immediately.
Maybe this test made sense.
Or we can add another test that reproduces this condition in a more controlled way